public/app/percona/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate.tsx

```diff
@@ -1,14 +1,15 @@
 /* eslint-disable react/display-name */
 import { logger } from '@percona/platform-core';
+import moment from 'moment/moment';
```

Comment:
we use moment and i also see date-fns, maybe we should move away from moment completely over time, given that the moment is no longer supported by developers? https://momentjs.com/docs/#:~:text=We%20now%20generally%20consider%20Moment%20to%20be%20a%20legacy%20project%20in%20maintenance%20mode.%20It%20is%20not%20dead%2C%20but%20it%20is%20indeed%20done.

Reply:
Nice one. I removed it from Alerts, but forgot this one 👍